### ISSUE TITLE
Issue Exporter Woocmmerce Function don't working

### ISSUE THREAD

**Issue by angelitodiaz1** [woothemes/woocommerce#31064]
Please provide us with the information requested in this bug report. Without these details, we won't be able to fully evaluate this issue. 
Bug reports lacking detail, or for any other reason than to report a bug, may be closed without action.

**Prerequisites (mark completed items with an [x]):**
- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**
When trying to export the products, it is generated in an infinite loop.

**Expected behavior**
Generate a CSV file with the total information of the store's products

**Actual behavior**
Loop infinite.
[Mon Nov 01 18:36:43.726310 2021] [:error] [pid 783707:tid 47631374292736] [client 190.78.90.64:28122] client denied by server configuration: /home/arflexco/public_html/wp-admin/admin-ajax.php, referer: https://arflex.com.ar/wp-admin/edit.php?post_type=product&page=product_exporter

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Go to '...'   Woocommerce - Products - 
2. Click on '....' Products Exporter

**Screenshots**
![1](https://user-images.githubusercontent.com/6534435/139746141-71fce40b-0565-4cd3-ab1d-1038b16c2519.png)
![2](https://user-images.githubusercontent.com/6534435/139746145-d17c0954-083e-4767-b1f2-eca8c85e8735.png)

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it here.
```
</details>
`
### WordPress Environment ###

WordPress address (URL): https://arflex.com.ar
Site address (URL): https://arflex.com.ar
WC Version: 5.8.0
REST API Version: ✔ 5.8.0
WC Blocks Version: ✔ 5.9.1
Action Scheduler Version: ✔ 3.3.0
WC Admin Version: ✔ 2.7.2
Log Directory Writable: ✔
WP Version: 5.8.1
WP Multisite: –
WP Memory Limit: 1 GB
WP Debug Mode: –
WP Cron: –
Language: es_AR
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.24
PHP Post Max Size: 50 MB
PHP Time Limit: 60000
PHP Max Input Vars: 80000
cURL Version: 7.71.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 10.3.31-MariaDB-cll-lve
Max Upload Size: 50 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.8.0
WC Database Prefix: DDG4rge_
Tamaño total de la base de datos: 861.66MB
Tamaño de los datos de la base de datos: 843.83MB
Tamaño del índice de la base de datos: 17.83MB
DDG4rge_woocommerce_sessions: Datos: 0.05MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_woocommerce_api_keys: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_woocommerce_attribute_taxonomies: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_woocommerce_downloadable_product_permissions: Datos: 0.02MB + Índice: 0.06MB + Motor InnoDB
DDG4rge_woocommerce_order_items: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_woocommerce_order_itemmeta: Datos: 0.05MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_woocommerce_tax_rates: Datos: 0.02MB + Índice: 0.06MB + Motor InnoDB
DDG4rge_woocommerce_tax_rate_locations: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_woocommerce_shipping_zones: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_woocommerce_shipping_zone_locations: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_woocommerce_shipping_zone_methods: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_woocommerce_payment_tokens: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_woocommerce_payment_tokenmeta: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_woocommerce_log: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_actionscheduler_actions: Datos: 1.52MB + Índice: 1.14MB + Motor InnoDB
DDG4rge_actionscheduler_claims: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_actionscheduler_groups: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_actionscheduler_logs: Datos: 1.52MB + Índice: 0.59MB + Motor InnoDB
DDG4rge_aiowps_events: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_aiowps_failed_logins: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_aiowps_global_meta: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_aiowps_login_activity: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_aiowps_login_lockdown: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_aiowps_permanent_block: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_cf_forms: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_cf_form_entries: Datos: 0.02MB + Índice: 0.06MB + Motor InnoDB
DDG4rge_cf_form_entry_meta: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_cf_form_entry_values: Datos: 0.02MB + Índice: 0.05MB + Motor InnoDB
DDG4rge_cf_pro_messages: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_cf_queue_failures: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_cf_queue_jobs: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_commentmeta: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_comments: Datos: 0.02MB + Índice: 0.09MB + Motor InnoDB
DDG4rge_e_submissions: Datos: 0.02MB + Índice: 0.27MB + Motor InnoDB
DDG4rge_e_submissions_actions_log: Datos: 0.02MB + Índice: 0.11MB + Motor InnoDB
DDG4rge_e_submissions_values: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_jet_cct_ordenes_prepedidos: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_post_types: Datos: 0.03MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_reviews: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_review_comments: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_review_guests: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_review_meta: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_review_types: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_smart_filters_indexer: Datos: 0.11MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_jet_taxonomies: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_links: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_ms_snippets: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_options: Datos: 5.48MB + Índice: 0.34MB + Motor InnoDB
DDG4rge_postmeta: Datos: 485.55MB + Índice: 10.03MB + Motor InnoDB
DDG4rge_posts: Datos: 344.52MB + Índice: 2.63MB + Motor InnoDB
DDG4rge_rank_math_internal_links: Datos: 0.05MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_rank_math_internal_meta: Datos: 0.05MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_snippets: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_termmeta: Datos: 0.22MB + Índice: 0.19MB + Motor InnoDB
DDG4rge_terms: Datos: 0.09MB + Índice: 0.09MB + Motor InnoDB
DDG4rge_term_relationships: Datos: 1.52MB + Índice: 0.30MB + Motor InnoDB
DDG4rge_term_taxonomy: Datos: 0.09MB + Índice: 0.11MB + Motor InnoDB
DDG4rge_usermeta: Datos: 0.11MB + Índice: 0.09MB + Motor InnoDB
DDG4rge_users: Datos: 0.02MB + Índice: 0.05MB + Motor InnoDB
DDG4rge_wc_admin_notes: Datos: 0.05MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wc_admin_note_actions: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_wc_category_lookup: Datos: 0.05MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wc_customer_lookup: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wc_download_log: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wc_order_coupon_lookup: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wc_order_product_lookup: Datos: 0.02MB + Índice: 0.06MB + Motor InnoDB
DDG4rge_wc_order_stats: Datos: 0.02MB + Índice: 0.05MB + Motor InnoDB
DDG4rge_wc_order_tax_lookup: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wc_product_meta_lookup: Datos: 1.52MB + Índice: 0.78MB + Motor InnoDB
DDG4rge_wc_reserved_stock: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wc_tax_rate_classes: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_wc_webhooks: Datos: 0.02MB + Índice: 0.02MB + Motor InnoDB
DDG4rge_wsal_metadata: Datos: 0.05MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wsal_occurrences: Datos: 0.02MB + Índice: 0.03MB + Motor InnoDB
DDG4rge_wt_iew_action_history: Datos: 0.06MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wt_iew_cron: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wt_iew_ftp: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB
DDG4rge_wt_iew_mapping_template: Datos: 0.02MB + Índice: 0.00MB + Motor InnoDB

### Post Type Counts ###

attachment: 2677
custom_css: 1
customize_changeset: 2
e-landing-page: 3
elementor_library: 20
jet-engine: 9
jet-engine-booking: 1
jet-smart-filters: 6
jet-theme-core: 1
jet-woo-builder: 12
listas-precios: 154
nav_menu_item: 13
page: 23
post: 3
product: 1120
product_variation: 3956
revision: 5136
shop_order: 18

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (25) ###

Editor clásico: por Colaboradores de WordPress – 1.6.2
Code Snippets: por Code Snippets Pro – 2.14.2
Content Control: por Code Atlantic – 1.1.9
Join.chat: por Creame – 4.1.15
DynamicConditions: por RTO GmbH – 1.5.1
Elementor Pro: por Elementor.com – 3.4.2
Elementor: por Elementor.com – 3.4.7
EmbedPress: por WPDeveloper – 3.2.1
If Menu - Visibility control for menu items: por Layered – 0.16.2
JetBlocks For Elementor: por Crocoblock – 1.2.10
JetCompareWishlist For Elementor: por Crocoblock – 1.4.0
JetElements For Elementor: por Crocoblock – 2.5.9
JetEngine - Custom visibility conditions: por Crocoblock – 1.0.0
JetEngine: por Crocoblock – 2.9.4
Jet Hide Empty Items: por Crocoblock – 1.0.0
JetMenu: por Crocoblock – 2.1.3
JetSmartFilters: por Crocoblock – 2.3.2
JetTabs For Elementor: por Crocoblock – 2.1.13
JetWooBuilder For Elementor: por Crocoblock – 1.10.5
JetProductGallery For Elementor: por Crocoblock – 2.0.4
User Role Editor: por Vladimir Garagulya – 4.60.2
Checkout Field Editor for WooCommerce: por ThemeHigh – 1.5.0
Mercado Pago payments for WooCommerce: por Mercado Pago – 5.5.0
WooCommerce: por Automattic – 5.8.0
WPFront Scroll Top: por Syam Mohan – 2.0.7.08086

### Inactive Plugins (1) ###

OoohBoi Steroids for Elementor: por OoohBoi – 1.8.0

### Dropin Plugins (1) ###

maintenance.php: maintenance.php

### Settings ###

API Enabled: –
Force SSL: –
Currency: ARS ($)
Currency Position: left
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Casa matriz de la tienda: #30 - /tienda/
Carrito: #31 - /carro/
Finalizar compra: #32 - /finalizar-comprar/
Mi cuenta: #33 - /mi-cuenta/
Términos y condiciones: ❌ Página no definida

### Theme ###

Name: SocialBuey Institucional
Version: 1.0.0
Author URL: https://www.socialbuey.com
Child Theme: ✔
Parent Theme Name: Hello Elementor
Parent Theme Version: 2.4.1
Parent Theme Author URL: https://elementor.com/?utm_source=wp-themes&utm_campaign=author-uri&utm_medium=wp-dash
WooCommerce Support: ✔

### Templates ###

Overrides: /home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/cart/cart.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/checkout/form-checkout.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/checkout/thankyou.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/dashboard.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/form-edit-account.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/form-login.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/my-account.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/my-address.php
/home/arflexco/public_html/wp-content/plugins/jet-woo-builder/templates/woocommerce/myaccount/orders.php

### Action Scheduler ###

Cancelada: 3
Oldest: 2021-11-01 11:48:50 -0300
Newest: 2021-11-01 11:48:50 -0300

Completar: 4.552
Oldest: 2021-10-02 10:01:48 -0300
Newest: 2021-11-01 17:23:28 -0300

Fallida: 3
Oldest: 2021-10-22 11:22:13 -0300
Newest: 2021-10-22 11:24:16 -0300

### Status report information ###

Generated at: 2021-11-01 19:34:07 -03:00
`

**Comment by tammullen** [comment]
Hi @angelitodiaz1 

Thank you for submitting the issue. However, I can’t reproduce it using the steps you provided. Everything is working as expected on my end and I was able to export products

![Screenshot 2021-11-03 at 21 24 07](https://user-images.githubusercontent.com/24649833/140195090-d3ceadb0-772f-471c-91b3-40aa8c8825d8.png)


![Screenshot 2021-11-03 at 21 24 42](https://user-images.githubusercontent.com/24649833/140195107-240c9bed-2c3d-442d-b6bc-4aca1fda63cf.png)

![Screenshot 2021-11-03 at 21 23 58](https://user-images.githubusercontent.com/24649833/140195121-c7e591ff-bc6c-4567-877b-e9a90758eb12.png)

The error might be caused by Apache web server configuration. You can learn more about it here. You may need to contact your host to resolve this. https://wiki.apache.org/httpd/ClientDeniedByServerConfiguration.

**Comment by angelitodiaz1** [comment]
Thanks so much.  Hosting support validated and now working.

**Comment by tammullen** [comment]
Thanks for letting us know! Glad it is working now! Closing this issue as no further action needed
